src/simulate.js

```diff
@@ -53,7 +52,37 @@ module.exports = function (opts, simHostOpts) {
     });
 };
 
-function parseStartPage(projectRoot) {
+var closeServer = function() {
+    return server.server && server.server.close();
+};
+
+var launchBrowser = function(target, url) {
+    return cordovaServe.launchBrowser({ target: target, url: url });
+};
+
+var simulate = function(opts) {
+    
+    var target = opts.target || 'chrome';
+    var simHostUrl;
+
+    return launchServer(opts)
```

Comment:
Do we still need `launchServer()` separate from `simulate()`? Or, if we do keep them separate (which I'm fine with), we don't need to have the `catch` handler in both places.